news/exit.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Added two ways of exit to the "Bash to xsh" article.
```

Comment:
FYI, we generally don't require news items for docs, but thank you!

Reply:
Thank you for info!